Cargo.toml

```
@@ -84,3 +84,7 @@ debug = true
 
 [profile.bench]
 debug = true
+
+[patch.crates-io]
+zcash_note_encryption = { git = "https://github.com/daira/librustzcash.git", rev = "f07c4c4358f2d062e2ce9108069927bcf16117a8" }
+group = { git = "https://github.com/zkcrypto/group.git", rev = "bbd331f564a05d516d7dd81a1ceb0e431d264546" }
```

```suggestion
group = { git = "https://github.com/zkcrypto/group.git", rev = "a7f3ceb2373e9fe536996f7b4d55c797f3e667f0" }
```
(This is the merge commit. bbd331f was an intermediate commit, even though it did include the necessary changes. )